plugins/inputs/mongodb/README.md

```diff
@@ -10,6 +10,7 @@
   ##   mongodb://10.10.3.33:18832,
   ##   10.0.0.1:10000, etc.
   servers = ["127.0.0.1:27017"]
+  gather_db_stats = false
```

Comment:
please change to gather_**per**db_stats, so it's clear what this option is doing